src/main/webapp/adminOnly/AdminOnly.js

```diff
@@ -13,13 +13,10 @@ import { PREFERRED_IRB } from "../util/TypeDescription";
 import { INITIAL_REVIEW } from "../util/TypeDescription";
 import { InputTextList } from "../components/InputTextList";
 import { Fundings } from "../components/Fundings";
-import { Spinner } from "../components/Spinner";
-import { spinnerService } from "../util/spinner-service";
 import { AlertMessage } from "../components/AlertMessage";
+import LoadingWrapper from "../components/LoadingWrapper";
 
-const ADMIN_ONLY_SPINNER = 'adminOnlySpinner';
-
-export const AdminOnly = hh(class AdminOnly extends Component {
+const AdminOnly = hh(class AdminOnly extends Component {
```

Comment:
Not a request for this PR, but I think an alternate syntax for this might look like:
```
export default LoadingWrapper(hh(class AdminOnly extends Component {... blah ... }), 'fixedTop') as AdminOnly
```
Might be worth looking at whether that saves boilerplate or makes it worse ... not really sure at this point.

Reply:
I discussed this with the team, we think it's more legible the way it is now. :smiley: